.github/workflows/pr.yml

```
@@ -100,3 +100,16 @@ jobs:
           fetch-depth: 0
       - name: Check if dockerimage build is working
         run: docker build -f ./Dockerfile .
+  gitlint:
+    name: Run gitlint checks
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Install gitlint into container
+        run: python -m pip install gitlint
+      - name: Run gitlint check
+        run: gitlint --config .gitlint --contrib contrib-title-conventional-commits,CC1 --commits origin/${{ github.event.pull_request.base.ref }}..HEAD
```

Missing new line at the end.

Also, as per the comment below:
```suggestion
        run: gitlint --commits origin/${{ github.event.pull_request.base.ref }}..HEAD
```
I also removed the `--config` option as we are using the default.